README.md

```diff
@@ -170,17 +170,17 @@ Here is an example of 'SendFunds' transaction:
 
 ```javascript
 var SendFunds = Exonum.newMessage({
```

Comment:
Could you please add a unit test for this snippet (and possibly other examples from the README) to ensure that it actually works?

Reply:
Done: https://github.com/exonum/exonum-client/pull/60/files#diff-7130f0fbba2575608a871f35ee60d08a